tasks/process_agent.py

```diff
@@ -58,23 +56,6 @@ def build(ctx, race=False, go_version=None, incremental_build=False,
         "BuildDate": datetime.datetime.now().strftime("%Y-%m-%dT%H:%M:%S"),
     }
 
-    goenv = {}
-    # TODO: this is a temporary workaround to avoid the garbage collection issues that the process-agent+go1.11 have had.
-    # Once we have upgraded the go version to 1.12, this can be removed
-    if go_version:
-        lines = ctx.run("gimme {version}".format(version=go_version)).stdout.split("\n")
-        for line in lines:
-            for env_var in GIMME_ENV_VARS:
-                if env_var in line:
-                    goenv[env_var] = line[line.find(env_var)+len(env_var)+1:-1].strip('\'\"')
-        ld_vars["GoVersion"] = go_version
```

Comment:
Thoughts on keeping this block and the system_probe one  even if we're not using in CI ? It's been useful to quickly change the go version in the past.  Same

Reply:
sure, I'll restore it

Reply:
done